src/dagmc/dagmcmetadata.hpp

```diff
@@ -8,118 +8,317 @@
 
 class dagmcMetaData {
  public:
-  // Constructor
+  /**
+   * @brief Constructs a new DagmcMetadata object.
+   *
+   * @param dagmc A pointer to the DagMC instance associated with this metadata.
+   */  // Constructor
   dagmcMetaData(moab::DagMC* DAGptr, bool verbosity = false,
                 bool require_density_present = true);
 
-  // Destructor
+  /**
+   * @brief Default destructor for the dagmcMetadata class.
+   */
   ~dagmcMetaData() = default;
 
-  // load the dagmc properties into maps
+  /**
+   * @brief Loads the DagMC properties into internal data maps.
+   *
+   * This function reads the properties from the associated DagMC instance and
+   * stores them in internal data structures for efficient access.
+   */
```

Comment:
This repeated line is causing a build error.

Reply:
Indeed. Thanks